src/profiler.rs

```diff
@@ -773,7 +773,8 @@ impl GpuProfiler {
 
                     #[cfg(feature = "tracy")]
                     if let Some(tracy_scope) = scope.tracy_scope.take() {
-                        tracy_scope.upload_timestamp(start_raw as i64, end_raw as i64);
+                        tracy_scope.upload_timestamp_start(start_raw as i64);
+                        tracy_scope.upload_timestamp_end(end_raw as i64);
```

Comment:
It looks like these were separated because Tracy wants the inner scopes nested between them (ie with the time values always monotonically increasing)

Reply:
ohhhh. Thanks for pointing that out!

Reply:
that should do. but tbh I still haven't tested it actually
[`d1f9a2d` (#92)](https://github.com/Wumpf/wgpu-profiler/pull/92/commits/d1f9a2d9a7afab7130602cd66423dadc9fc3900f)